src/arr-crud.js

```diff
@@ -0,0 +1,65 @@
+import { dragndrop, renderTasks } from './drag-and-drop.js'; // eslint-disable-line
+import { checkboxesUpdate } from './status-updates.js'; // eslint-disable-line
+import { setStorage } from './index.js'; // eslint-disable-line
+import { seeMoreUpdate } from './status-updates.js'; // eslint-disable-line
+
+export class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export class ToDoList {
+  constructor(toDoArr = []) {
+    this.tasksArr = toDoArr;
+  }
```

Comment:
Task class is a data model, TodoList is a utility class, they shouldn't be together, I would kindly suggest you take a look at the single responsibility model and separate data from functionality.